README.adoc

```diff
@@ -139,7 +139,16 @@ NOTE: `fabric8-ui` is setup to do reloading and sourcemaps automatically when yo
 run `npm start`.
 
 ** To hit stack analysis api in standalone mode**
-Put a token in the environment variable with key as 'STACK_API_TOKEN'.
+
+Export below variables 
+
+```
+
+export ANALYTICS_RECOMMENDER_URL='https://recommender.api.openshift.io/'
+export ANALYTICS_LICENSE_URL='https://license-analysis.api.openshift.io/'
+export OSIO_AUTH_TOKEN='osio-token'
+
+```
```

Comment:
add: `where OSIO_AUTH_TOKEN is your PROD [OSIO](https://openshift.io/) auth token (login to the app and inspect to retrieve token)`